.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+on:
+  push:
+    branches:
+      - develop
```

Comment:
```suggestion
```
No need to run sonar after PR merge on develop branch

Reply:
I think it is necessary, as we need  to have a view on current sonar status on develop. So I prefer to let it on develop